tests/org/intellij/erlang/resolve/ErlangFunctionResolutionTest.java

```diff
@@ -16,48 +16,86 @@
 
 package org.intellij.erlang.resolve;
 
+import com.intellij.openapi.projectRoots.Sdk;
 import com.intellij.psi.PsiElement;
 import com.intellij.psi.PsiFile;
 import com.intellij.psi.PsiReference;
 import com.intellij.psi.util.PsiTreeUtil;
+import com.intellij.testFramework.LightProjectDescriptor;
+import com.intellij.testFramework.fixtures.DefaultLightProjectDescriptor;
 import com.intellij.util.ArrayUtil;
-import org.intellij.erlang.psi.ErlangFile;
-import org.intellij.erlang.psi.ErlangFunction;
-import org.intellij.erlang.psi.ErlangFunctionCallExpression;
+import org.intellij.erlang.psi.*;
+import org.intellij.erlang.psi.impl.ErlangFunctionReferenceImpl;
+import org.intellij.erlang.psi.impl.ErlangPsiImplUtil;
+import org.intellij.erlang.sdk.ErlangSdkRelease;
+import org.intellij.erlang.sdk.ErlangSdkType;
 import org.intellij.erlang.utils.ErlangLightPlatformCodeInsightFixtureTestCase;
+import org.jetbrains.annotations.NotNull;
 
 public class ErlangFunctionResolutionTest extends ErlangLightPlatformCodeInsightFixtureTestCase {
   @Override
+  protected LightProjectDescriptor getProjectDescriptor() {
+    return new DefaultLightProjectDescriptor() {
+      @Override
+      public Sdk getSdk() {
+        return ErlangSdkType.createMockSdk("testData/mockSdk-R15B02/", ErlangSdkRelease.V_R15B02);
+      }
+    };
+  }
+
+  @Override
+  protected void setUp() throws Exception {
+    super.setUp();
+    setUpProjectSdk();
+  }
+
+  @Override
   protected String getTestDataPath() {
     return "testData/resolve/function/" + getTestName(true) + "/";
   }
 
-  protected void doTest(String focusedFilePath, String expectedFilePath, String... otherFilePaths) {
+  public void testNoAutoImport()               { doFunctionCallTest("test.erl", "incl.erl"); }
+  public void testNoAutoImportWithTuple()      { doFunctionCallTest("test.erl", "incl.erl"); }
+  public void testPreferImportFromUserModule() { doFunctionCallTest("test.erl", "incl.erl"); }
+  public void testPreferLocalFunction()        { doFunctionCallTest("test.erl", "test.erl", "incl.erl"); }
+  public void testPreferFirstImport()          { doFunctionCallTest("test.erl", "incl.erl", "another_incl.erl"); }
+
+  public void testGetArityFromParameter()      { doParameterTest("test.erl", "test.erl"); }
+  public void testGetModuleFromParameter()     { doParameterTest("test.erl", "module.erl"); }
+  //This test needs ErlangSpecificationStub
+  //public void testFunParameterInSpawnMonitor() { doParameterTest("test.erl", "test.erl"); }
```

Comment:
This test does not need ErlangSpecificationStub. 

What it needs is a correct implementation of ```org.intellij.erlang.psi.impl.ErlangFileImpl#getSpecifications``` (which, in turn, can be done via stubs).

Use ```//TODO do something``` comments to mark things which are to be done later.

BTW, "This test needs ErlangSpecificationStub" doesn't say what should be done with the commented code.